### DISCUSSION TITLE
Digital Products

### DISCUSSION THREAD

**Discussion by asesmero** [microweber/microweber#1013] [Q&A]
Is anyway to make downloadable products on the shop?

**Comment by VeniceInventors** [comment]
If the functionality doesn't already exists in Microweber, I may be able to help with that.
I'm new to this framework, literally found and installed Mw for the first time today, but I have plenty of experience with digital products delivery in Perl/PHP so I could cook something up once I'm a bit more familiar with Mw.
Just remind me if you don't hear back from me, as I get sidetracked with other projects.
